spec.bs

```diff
@@ -875,19 +875,15 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
       1. Set |auctionConfig|'s [=auction config/seller signals=] to the result of
         [=serializing a JavaScript value to a JSON string=], given |result|.
     * To handle an error, set |auctionConfig|'s [=auction config/seller signals=] to failure.
-1. If |config|["{{AuctionAdConfig/directFromSellerSignals}}"] [=map/exists=]:
-  1. TODO: The receiving end of this isn't specified yet, so there is no place to put the computed value.
-  1. [=Handle an input promise in configuration=] given |auctionConfig| and |config|["{{AuctionAdConfig/directFromSellerSignals}}"]:
+1. If |config|["{{AuctionAdConfig/directFromSellerSignalsHeaderAdSlot}}"] [=map/exists=]:
+  1. [=Handle an input promise in configuration=] given |auctionConfig| and |config|["{{AuctionAdConfig/directFromSellerSignalsHeaderAdSlot}}"]:
     * To parse the value |result|:
-      1. Let |directFromSellerSignalsPrefix| be the result of running the [=URL parser=] on |result|.
-      1. [=exception/Throw=] a {{TypeError}} if any of the following conditions hold:
-        * |directFromSellerSignalsPrefix| is failure;
-        * |directFromSellerSignalsPrefix| is not [=same origin=] with |auctionConfig|'s
-          [=auction config/seller=];
-        * |directFromSellerSignalsPrefix|'s [=url/query=] is not null.
-      1. [=Assert=]: |directFromSellerSignalsPrefix|'s [=url/scheme=] is "`https`".
+      1. If |result| is not undefined or null, and typeof |result| is [=string=]:
+        1. Set |auctionConfig|'s [=auction config/direct from seller signals header ad slot=] to |result|.

```

Comment:
In the implementation, the auction config doesn't have the actual adSlot string; this is always passed via the promise resolution path, but I'm not sure how much it matters since the user-visible behavior is the same?

Reply:
I think the line above (879) handles the promise, like how other promises in auction config is handled?
Here is when the promised is resolved with the actual string.

Reply:
Can this be resolved or is there some outstanding confusion/comment here?

Reply:
Yes, I think we can close this.